Misc/NEWS.d/next/Library/2020-02-23-02-09-03.bpo-38691.oND8Sk.rst

```diff
@@ -0,0 +1 @@
+Updated :meth:`importlib._bootstrap_external._make_relax_case` to ignore :envvar:`PYTHONCASEOK` when -E or -I flags are used.
```

Comment:
I would prefer to avoid mentioning a private module (_bootstrap_external) in a public Changelog entry. Reuse the text that you wrote above:

```
  The :mod:`importlib` module now ignores the :envvar:`PYTHONCASEOK`
  environment variable when the :option:`-E` or :option:`-I` command line
  options are being used.
```

Reply:
@vstinner Fixed.